### PR TITLE
"layout_moyo-header" PR

### PR DESCRIPTION
**Pull request by Rocketjet**
All tests passed.

### CODE REVIEW

**Comment by Rocketjet** [comment]
- [DEMO LINK](https://Rocketjet.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Rocketjet.github.io/layout_moyo-header/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (herkoss) [inline src/index.html:7]:
```suggestion
    <meta
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <header class="header">
+      <img class="logo" src="images/logo.png"
+        alt="logo">
```
Comment (herkoss) [inline src/index.html:16]:
```suggestion
      <img
        class="logo"
        src="images/logo.png"
        alt="logo"
      >
```

Comment (herkoss) [reply src/index.html:16]: Fix code style here and in a whole file

Comment (herkoss) [reply src/index.html:16]: It must be a clickable link

**Review by herkoss — CHANGES_REQUESTED** [review]
Well done!👍
But it needs some refactoring)
Good luck🍀

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,97 @@
...
+  </head>
+
+  <body>
+    <header class="header">
+      <a href="#">
+        <img
+        class="logo"
+        src="images/logo.png"
+        alt="logo"
+      >
```
Comment (herkoss) [inline src/index.html:28]:
```suggestion
        <img
          class="logo"
          src="images/logo.png"
          alt="logo"
        >
```

**Review by herkoss — APPROVED** [review]
Good job!👏👍

**Pull request closed without merging**